tests/foreman/cli/test_import.py

```diff
@@ -1226,7 +1246,7 @@ def test_bz1160847_translate_macros(self):
             u'channel_id': u'3',
             u'channel': u'config-1',
             u'channel_type': u'normal',
-            u'path': u'/etc/sysconfig/rhn/systemid',
+            u'path': gen_string('utf8') + gen_string('alphanumeric'),
```

Comment:
the path does not matter in this case. It is being converted to a single file name,
sp e.g. ```u'/etc/sysconfig/rhn/systemid'``` will become `etc_system_sysconfig_rhn_systemid.erb`
there is a rule for removing 'ugly' characters (```[^\da-zA-Z\-\.\_]```)
so the test is just to take some string containing `nice` and `ugly` characters.